standards/certification/digisov-and-cert.md

```diff
@@ -0,0 +1,121 @@
+we differentiate between several levels of digital sovereignty.
+We'll skip stage 0, introduced by Gregor Schuhmacher in his description, which
+specifies using a cloud at all as the pre-step to be taken. This has relevance,
+as some companies continue to call solutions that are not on-demand, not
+self-service API driven, not metered
+(see [NIST definition of cloud](https://nvlpubs.nist.gov/nistpubs/legacy/sp/nistspecialpublication800-145.pdf))
+to be (private) clouds. We talk about real clouds, where deployment of infrastructure
+is API-driven, unlocking DevOps teams productivity.
+
+The levels as seen by the SCS movement are:
+
+1. Control over data and data sharing and ability to fulfill regulatory requirements (GDPR)
+2. Capability to chose between *highly compatible* operators, this way enabling a provider
+   switch or using several providers in a federated fashion. This also includes the
+   possibility to run your infrastructure in a *highly compatible* manner.
+3. Capability to influence and shape the infrastructure, enabling innovation at the
+   infrastructure layer.
+4. Transparency over operational aspects of running infrastructure, this way supporting
+   to overcome a skill gap to being able to operate infrastructure in a highly reliable
+   manner.
+
+These aspects of sovereignty drive the work from the SCS team.
+
+Level number 1 is sometimes referred to as "data sovereignty". Achieving it does require
+cloud infrastructure and cloud operations that can not be interfered with by actors that
+are outside of the respective jurisdiction. For Europeans that need to observe GDPR, this
+excludes using US clouds for personally identifiable information, expecting that the
+adequacy decisions for the US do not fully address the risks. The SCS project does not
+have deep legal expertise and refers to the work from [noyb](https://noyb.eu/)
+and [ENISA](https://www.enisa.europa.eu/) here.
+
+In order to achieve level 2,
+the SCS community has worked on standards that define the APIs and the infrastructure
+behavior, so application developers and application operators can deploy the same application
+using the same automation and rely on the same infrastructure behavior to operate the
+application in a resilient way. The standards allow for switching providers or to use
+several providers in a federated way. Operating own infrastructure according to the same
+standards is also possible, allowing for hybrid cloud setups without technical barriers.
+
+Level 3 drives the work on a comprehensive openly developed open source software stack,
+allowing operators to use, study, change and redistribute the software according to the
+[Four Freedoms](https://en.wikipedia.org/wiki/The_Free_Software_Definition) of free software. We are requiring
+a complete stack that uses real open source licenses (as defined by [OSI](https://opensource.org/))
```

Comment:
“real” has no real meaning, remove it and instead make the OSI reference even more clear.

```suggestion
a complete stack that uses [OSI](https://opensource.org/)-approved open source licenses
```